homeassistant/components/trafikverket_weatherstation/__init__.py

```diff
@@ -30,7 +30,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     title = entry.title
     if unload_ok:
-        del hass.data[DOMAIN][entry.entry_id]
         if not hass.data[DOMAIN]:
```

Comment:
We don't use hass.data at all in this integration so we can remove all of that both here and in `async_setup_entry`.